src/Components/Card/TeamMemberCard/TeamMember.js

```diff
@@ -0,0 +1,64 @@
+import empty from '../../../Assets/Images/empty.png';
+import bassam from '../../../Assets/Images/bassam.png';
+import jana from '../../../Assets/Images/jana.png';
+import osama from '../../../Assets/Images/osama.png';
+import ibrahim from '../../../Assets/Images/ibrahim.png';
+
+const card = [
+  {
+    photo: <img src={osama} alt="programmerIMG" />,
+    name: 'Osama Abdullah',
+    role: 'Lead Engineer',
+  },
+  {
+    photo: <img src={empty} alt="programmerIMG" />,
+    name: 'Abdellatif Edlby',
+    role: 'Junior Engineer',
+  },
+  {
+    photo: <img src={jana} alt="programmerIMG" />,
+    name: 'Jana Khanji',
+    role: 'Junior Engineer',
+  },
+  {
+    photo: <img src={bassam} alt="programmerIMG" />,
+    name: 'Bassam Edelbi',
+    role: 'Junior Engineer',
+  },
+  {
+    photo: <img src={ibrahim} alt="programmerIMG" />,
+    name: 'Ibrahim Nour',
+    role: 'Junior Engineer',
+  },
+
+  {
+    photo: <img src={empty} alt="programmerIMG" />,
+    name: 'Tommy Chaanin',
+    role: 'Junior Engineer',
+  },
+].map((anObjectMapped, index) => {
+  return (
+    <div className="w-52 m-auto " key={`${anObjectMapped.name}`}>
+      {anObjectMapped.photo}
+      <div className="flex justify-center text-center">
```

Comment:
can you make it more interesting like the name is a bit bigger and the role is grey-ish color, maybe add hover effect on the image, you know its pretty simple right now and it would be nice to have more things to it (: